ui/packages/consul-ui/app/templates/dc/services/instance/upstreams.hbs

```diff
@@ -35,6 +35,15 @@ as |route|>
           @filter={{filters}}
         />
       {{/if}}
+      {{#if (eq proxy.ServiceProxy.Mode 'transparent')}}
+        <Consul::Notice
+          @type="warning"
+          @for="tproxy-mode"
+          @link="{{env 'CONSUL_DOCS_URL'}}/connect/transparent-proxy"
+          @internal={{false}}
+          @action={{true}}
+        />
+      {{/if}}
```

Comment:
I'd prefer to use a normal `<Notice />` component here, (then we keep the `h3` etc in the top level route template) would it be ok to change that out?

Reply:
@johncowen Updated to use `<Notice /> `